src/cavegen.h

```diff
@@ -150,18 +154,16 @@ class CavesRandomWalk {
 
 	// ndef is a mandatory parameter.
 	// If gennotify is NULL, generation events are not logged.
-	CavesRandomWalk(INodeDefManager *ndef,
-		GenerateNotifier *gennotify = NULL,
-		s32 seed = 0,
-		int water_level = 1,
-		content_t water_source = CONTENT_IGNORE,
-		content_t lava_source = CONTENT_IGNORE);
+	CavesRandomWalk(INodeDefManager *ndef, GenerateNotifier *gennotify = NULL,
+			s32 seed = 0, int water_level = 1,
+			content_t water_source = CONTENT_IGNORE,
+			content_t lava_source = CONTENT_IGNORE);
```

Comment:
2-tab too?
The problem here is that we have to tell the formatter one of them...